### DISCUSSION TITLE
Docker folder

### DISCUSSION THREAD

**Discussion by MichaelLambert78** [tteck/Proxmox#3066] [Help Needed] [answered]
I've used a bunch of the built in container scripts to create containers, and now I'm trying to get resources from that container that reference the docker folder. I can't seem to find it and could use a point in the right direction.

**Comment by tteck** [comment]
This `/var/lib/docker/`?

**Comment by MichaelLambert78** [comment]
Correct, none of my containers seem to have that folder 
![Screenshot 2024-05-25 182106](https://github.com/tteck/Proxmox/assets/12418576/f23e54a1-a297-4397-ad3c-424b733abafb)

**Reply by tteck** [reply, marked as answer]
Maybe none of your containers are using Docker.

**Comment by MichaelLambert78** [comment]
ahh, my bad, I'm using an LXC container not Docker. Trying to get the individual containers setup in Uptime-Kuma
